src/js/api/auth/login.test.js

```diff
@@ -0,0 +1,28 @@
+import { login } from "./login.js";
+import localStorageMock from "../../../../mocks/localStorage.mock.js";
+import userMock from "../../../../mocks/user.mock.js";
+
+const USER_DATA = userMock;
```

Comment:
```suggestion
import USER_DATA from "../../../../mocks/user.mock.js";
```

It's a default export, so you can skip the assignment and just name the import whatever you want.